core/jazz_es-kinesis-log-streamer/components/utils.js

```diff
@@ -20,13 +20,33 @@
   @version: 1.0
 **/
 
+const AWS = require('aws-sdk');
 const crypto = require('crypto');
 const logger = require("../components/logger.js");
 const config = require("../config/global_config.json")
 var truncate = require('unicode-byte-truncate');
 
 // Helper functions
 
+function getLogsGroupsTags(logGroupName) {
+  var cloudwatchlogs = new AWS.CloudWatchLogs();
```

Comment:
This will work for logs coming in from primary account but will break for logs coming in from non-primary AWS accounts/regions. Since, we stream logs to single kinesis stream, simple look up like this based on logGroupName might give you 400 if you are dealing with logGroup from a different account/region. 

We need to figure out the correct account/region, assume the corresponding role and make this call (metrics API does something similar).

Reply:
Done